static/css/stat_var_hierarchy.scss

```diff
@@ -21,39 +21,48 @@ $medium-font-size: 0.85rem;
 $small-font-size: 0.75rem;
 
 .stat-var-hierarchy-container {
-  min-height: 20rem;
   font-size: $medium-font-size;
+  font-size: 14px;
+  font-weight: 700;
   line-height: 1.8rem;
+  line-height: 28px;
```

Comment:
remove the old font-size and line-height?

Reply:
Good catch, removed the old one.

Reply:
there are still 2 font-sizes?

Reply:
Fixed!